Code/src/main/java/nl/utwente/group10/ui/menu/CircleMenu.java

```diff
@@ -57,6 +57,7 @@ public CircleMenu(Block block) {
 
         // Animation
         this.setAnimationInterpolation(CircularPane::animateOverTheArcWithFade);
+        this.setAnimationDuration(Duration.ONE);
```

Comment:
> Does setting the interpolation make sense if the animation is only 1ms long?

Yes because without setting it the menu will not show at all, a quirky behaviour from the library.